src/components/ui/tabs/component.tsx

```diff
@@ -0,0 +1,53 @@
+import clsx from 'clsx'
+import type { FC, ReactNode, SetStateAction } from 'react'
+import { createContext, useState } from 'react'
+import styles from './styles.module.scss'
+
+export const TabsContext = createContext({
+  active: 0,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  setActive: (i: SetStateAction<number>) => {},
+})
+
+export const Tabs = ({ className, items, children: Component }: TProps) => {
```

Comment:
Прикольный компонент            
```suggestion
export const Tabs = ({ className, items, children: Component }: TProps) => {
```

Reply:
Ага